app/screens/channel/channel_nav_bar/channel_nav_bar.js

```diff
@@ -70,6 +70,14 @@ export default class ChannelNavBar extends PureComponent {
         }
     };
 
+    drawerButtonVisible = () => {
+        if (DeviceTypes.IS_TABLET && Platform.OS === 'ios') {
+            return false;
+        }
+
+        return (!this.state.permanentSidebar || !this.state.isSplitView);
```

Comment:
```suggestion
        if (Platform.OS === 'android') {
            return true;
        }

        return !DeviceTypes.IS_TABLET || this.state.isSplitView || !this.state.permanentSidebar;
```

Reply:
This is (way) simpler, and therefore better. Done (with needed tests) in 8855aa734a2269df9b7be40544028f269ebe7e05